manifests/init.pp

```diff
@@ -40,7 +44,9 @@
   Array $whitelist = ['127.0.0.1/8', '192.168.56.0/24'],
   Hash[String, Hash] $custom_jails = {},
   String[1] $banaction = 'iptables-multiport',
-) inherits ::fail2ban::params {
+) {
+  notify { "fail2ban will use ${config_file_template} as template": }
```

Comment:
This trigger a change, and with `--detailed-exitcodes` puppet will exit with 2 and not 0.

This is used in the test suite and seems to be the cause of the failure on the Beaker tests with CentOS I checked.

Instead of a resource, you can use the `notice()` function (always shown as far as I can recall), or maybe better `info()` (only shown when running with `--verbose`) or `debug()` (shown when running with `--debug`).

Reply:
OK, but the commit will be removed from final work. I put it here only to help user to test the branch.

Reply:
I did not find a similar "notification" in the code in the `master` branch so I just dropped that line in 839b6027f3c97fa190ba29f0164e14c165096d97.  It indeed fix the beaker integration tests :wink: